project_task_material/models/project.py

```diff
@@ -30,12 +29,3 @@ class ProjectTaskMaterial(models.Model):
     product_id = fields.Many2one(
         comodel_name='product.product', string='Product', required=True)
     quantity = fields.Float(string='Quantity')
-
-    @api.multi
-    @api.constrains('quantity')
-    def _check_quantity(self):
-        for material in self:
-            if not material.quantity > 0.0:
```

Comment:
Thanks for contribution,

I'm not sure what's the business logic to get negative task material quantity, any way you should probably add opt-in to let administrator allows or not negative quantity and forbid by defaut to avoid to break existing behaviour.

Could you fix test according if you want to get a chance to have your work merged ?